CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.5.12] - 2023-11-29
```

Comment:
I don't think the `data structured have been replaced` change is part of [1.5.12]

[List of commits which are in main which aren't part of 1.5.12](https://github.com/nhsconnect/integration-adaptor-gp2gp/compare/1.5.12...main)